.gitignore

```diff
@@ -1 +1,4 @@
 node_modules
+.idea
```

Comment:
It's better to properly ignore your editor files with [global ignore file](https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore) and keep `.gitignore` clean

Reply:
@viktor-yakubiv I didn't know about the existence of global ignore file but I still think it's better to keep it in local `.gitignore`. Actually, I would be even  more strict and generate automatic `.gitignore` file for all known IDEs and code related stuff like cache directories  `.npm-cache` ... It prevents all of the people who want to contribute to this repository to accidentally push unwanted stuff not just me. :)